providers/meta/media_test.go

```
@@ -0,0 +1,62 @@
+	tests := []struct {
+		name    string
+		size    int
+		wantErr bool
+	}{
+		{"at the cap", cap, false},
+		{"one byte over", cap + 1, true},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				w.Write(make([]byte, tt.size))
```

golangci-lint (2.13.2)

[error] Error return value of `w.Write` is not checked

(errcheck)